tests/integration.rs

```diff
@@ -972,8 +972,8 @@ fn check_latest_events_exact(latest_events: &[&MetaEvent], event_file: &str) ->
 }
 
 fn test_cpu_topology(threads_per_core: u8, cores_per_package: u8, packages: u8, use_fw: bool) {
-    let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
-    let guest = Guest::new(Box::new(focal));
+    let disk_config = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
```

Comment:
May be it would be better to create a global variable in this file with name like DEFAULT_SUPPORTED_IMAGE and assign it to either focal or jammy on top of this file. Then declare like follow
 let disk_config = UbuntuDiskConfig::new(DEFAULT_SUPPORTED_IMAGE.to_string());

Less change when we want to upgrade the image from one version to another.